tests/apps/courses/test_api_course_run_sync_edx.py

```diff
@@ -1233,3 +1233,203 @@ def test_api_course_run_sync_catalog_visibility_about(self, mock_signal):
             course_run.catalog_visibility, CourseRunCatalogVisibility.COURSE_ONLY
         )
         self.assertFalse(mock_signal.called)
+
+    # Bulk
+
+    @override_settings(
+        RICHIE_DEFAULT_COURSE_RUN_SYNC_MODE="sync_to_public", TIME_ZONE="utc"
+    )
+    def test_api_course_run_sync_create_bulk_success(self, mock_signal):
+        """
+        It should be possible to synchronize a list of course runs in bulk.
```

Comment:
You will have to move this test within `test_api_course_run_sync.py` file when you will update our branch.

Reply:
Done! Good catch :astonished: